ext/cl_khr_icd.asciidoc

```diff
@@ -60,6 +63,14 @@ members can be appended.
 Functions which do not have an argument from which the vendor implementation
 may be inferred have been deprecated and may be ignored.
 
+*New in version 2.0.0*, object that are ICD 2 compatible should also contain
+a `disp_data` field of type `void *` that the loader is free to modify for a
```

Comment:
Does it mean that the loader is supposed to set disp_data of all objects involved in the call to loader-specific dispatch data to transfer the state, but it doesn't have to?

Reply:
Only of the platforms, and only at initialization (and when I am done with the instance support, just after an instance platform is created).

Reply:
OK. It's not obvious from the text why this special casing for loader setting it for the platform only.